src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/ActiveDirectoryAuthenticationProvider.cs

```diff
@@ -145,9 +145,7 @@ public override async Task<SqlAuthenticationToken> AcquireTokenAsync(SqlAuthenti
                 DefaultAzureCredentialOptions defaultAzureCredentialOptions = new()
                 {
                     AuthorityHost = new Uri(authority),
-                    SharedTokenCacheTenantId = audience,
-                    VisualStudioCodeTenantId = audience,
-                    VisualStudioTenantId = audience,
+                    TenantId = audience,
```

Comment:
This is going to conflict with changes in #2380.

Reply:
So, from what I noticed that PR is going to address this issue. Thank you. I will close this PR after your confirmation.

Reply:
That would be easiest, yes.